radiation/python/radsw/radsw_param.py

```diff
@@ -0,0 +1,272 @@
+import numpy as np
+
+# Parameter constants for SW band structures
+
+# band range lower index
+nblow = 16
+# band range upper index
+nbhgh = 29
+# total number of SW bands (14)
+nbands = nbhgh - nblow + 1
+# total number of g-point in all bands
```

Comment:
what's a g-point?

Reply:
When numerically integrating over wavenumber, the spectrum is discretized in a particular way, and the discretization points are referred to as "g-points" by convention.  I remember learning this in radiation class, although I don't remember how they are computed